signing_service/README.md

```diff
@@ -34,10 +34,15 @@ python -m pytest -p no:django .
 
 To run SigningService you need the following:
 
-- Address and port of a Concent cluster to connect to. If port is omitted, default `9055` will be used.
+- Address and port of a MiddleMan service on a Concent cluster to which SigningService wants to connect to.
+  If port is omitted, default `9055` will be used.
 - A private key for signing sent messages.
 - Concent's public key for verifying signatures of received messages.
 - Ethereum private key for signing Ethereum transactions.
 - [Sentry DNS](https://docs.sentry.io/quickstart/#configure-the-dsn) if you want to submit error reports to a
-  Sentry instance (optional but recommended).
-    - If you want to submit reports to the instance used by Concent (recommended) please also use `--sentry-environment` option to set the `environment` tag to `concent-staging`, `concent-testnet` or `concent-mainnet`, depending on which cluster you're connecting to. If you're submitting to any other instance, you can set this tag to anything you want or even simply omit it. Its only purpose is to let us discern reports from different environments in Sentry.
+- Sentry instance (optional but recommended).
+    - If you want to submit reports to the instance used by Concent (recommended) please
+      also use `--sentry-environment` option to set the `environment` tag to `concent-staging`, `concent-testnet`
+      or `concent-mainnet`, depending on which cluster you're connecting to.
```

Comment:
When breaking lines please stick to one sentence per line. See https://github.com/Code-Poets/coding-style/wiki/markdown-style

I think this is the best compromise between readability (most lines are still short) and having sensible diffs (you don't have to rewrap the whole paragraph after every little change).

Reply:
Done :+1: I have rephrased this part a little bit and split it into several smaller sentences.